recipes-tag/intake-bluesky/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.0a8" %}
+{% set setup_py = load_setup_py_data() %}
+
+package:
+    name: 'intake-bluesky'
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/bluesky/intake-bluesky
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: pip install intake-bluesky
+    skip: True  # [py2k]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - versioneer
+
+    run:
+        - dask[bag]
```

Comment:
I don't know if this pip-derived subpackage selection is valid for conda recipe. Did it work with the local build?

Reply:
It may work. Because it touch the msgpack issue which is below dask[bag] line. Not 100% sure though

Reply:
It works. at least on docker